src/main/scala/com/databricks/spark/avro/ConfluentSparkAvroUtils.scala

```diff
@@ -21,15 +21,21 @@
  */
 package com.databricks.spark.avro
 
+import com.amazonaws.services.kms.model.DecryptRequest
+import com.amazonaws.services.kms.{ AWSKMS, AWSKMSClient }
+import javax.crypto.Cipher
+import javax.crypto.spec.SecretKeySpec
 import io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient
 import io.confluent.kafka.serializers.KafkaAvroDeserializer
+import java.nio.ByteBuffer
 import org.apache.avro.Schema
 import org.apache.commons.logging.LogFactory
 import org.apache.spark.sql.catalyst.expressions.GenericRow
 import org.apache.spark.sql.expressions.UserDefinedFunction
 import org.apache.spark.sql.functions.udf
 import org.apache.spark.sql.types.DataType
 
+import scala.collection.mutable.{Map => MutableMap}
```

Comment:
This is fine, but convention is to import `scala.collection.mutable`
and write `mutable.Map`